ui-components/__tests__/actions/ButtonGroup.test.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import { render, cleanup } from "@testing-library/react"
+import { Button } from "../../src/actions/Button"
+import { ButtonGroup } from "../../src/actions/ButtonGroup"
+import { AppearanceSizeType } from "../../src/global/AppearanceTypes"
+
+afterEach(cleanup)
```

Comment:
cleanup is no longer needed on jest tests https://testing-library.com/docs/react-testing-library/api/#cleanup